regression/daily/runPteTestSuite.sh

```diff
@@ -23,6 +24,16 @@ archivePTE() {
     fi
 }
 
+archiveBarebones() {
+    if [ ! -z $GERRIT_BRANCH ] && [ ! -z $WORKSPACE ]; then
+        # GERRIT_BRANCH is a Jenkins parameter and WORKSPACE is a Jenkins directory.This function is used only when the test is run in Jenkins to archive the log files.
+        echo "------> Archiving generated logs"
+        tCurr=`date +%m%d`
+        mkdir -p $WORKSPACE/archives/PTE_Test_Logs
+        cp $FabricTestDir/regression/barebones/barebones-pteReport.log $WORKSPACE"/archives/PTE_Test_Logs/barebones-pteReport-"$tCurr".log"
+    fi
+}
+
```

Comment:
There won't ever be called, GERRIT_BRANCH doesn't exist in the current Jenkin's infrastructure, this was legacy to Hyperledgers old Jenkins.

Reply:
will remove it